pkg/go/graph/graph_edge.go

```diff
@@ -35,7 +36,7 @@ func (n *AuthorizationModelEdge) Attributes() []encoding.Attribute {
 		})
 	}
 
-	if n.edgeType == RewriteEdge {
+	if n.edgeType == ComputedEdge {
```

Comment:
not for this PR but curious on why we don't use switch / case?

Reply:
No good reason 😆  i can send a follow-up